runtime/doc/lua.txt

```diff
@@ -1598,7 +1598,8 @@ show_pos({bufnr}, {row}, {col}, {filter})                     *vim.show_pos()*
 deep_equal({a}, {b})                                        *vim.deep_equal()*
     Deep compare values for equality
 
-    Tables are compared recursively unless they both provide the `eq` metamethod. All other types are compared using the equality `==` operator.
+    Tables are compared recursively unless they both provide the `eq`
+    metamethod. All other types are compared using the equality `==` operator.
```

Comment:
This is autogenerated by `scripts/gen_vimdoc.py` . 

That script would need to be fixed, else this will just get overwritten.